lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +241,72 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Distribution(object):
+    def __init__(self, name_string, version_string=None):
+        if not name_string.startswith(('svn+', 'git+', 'hg+', 'bzr+', 'file:')):
+            self._plain_distribution = True
+            if version_string:
+                version_string = version_string.lstrip()
+                separator = '==' if version_string[0].isdigit() else ' '
+                name_string = separator.join((name_string, version_string))
+            self._requirement = Requirement.parse(name_string)
+            self._distribution_name = self._requirement.name.lower()
+        else:
```

Comment:
If you could swap if and else parts, you'd avoid `not` negation in startswith check and also could omit `else` and dedent everything what's in if part now.

Reply:
I didn't get, how to omit `else`? We should only execute one of the two code blocks.